lib/commander/runner.rb

```diff
@@ -91,6 +91,18 @@ def version
       format('%s %s', program(:name), program(:version))
     end
 
+    ##
+    # Suppresses the --trace message for these classes
+    # However --trace will still display the backtrace if included
+
+    def suppress_trace_classes(*classes)
```

Comment:
Is there a need for this to take multiple classes? I think it would be simpler and clearer if there is just the option to have `Commander` suppress showing `--trace` for a single error class and its descendants.

This would also make things clearer in Metalware, as we would know there's only a single error class (with descendants) which this error message will not be shown for, and everything else it will be.

Reply:
Fixed